lib/utils/requireModule.js

```diff
@@ -1,25 +1,20 @@
 const path = require('path');
 
-const requireModule = module.exports = function(fullpath) {
+const requireModule = module.exports = function (fullpath) {
   let exported;
   try {
     exported = require(fullpath);
   } catch (err) {
-    if (err.code === 'MODULE_NOT_FOUND') {
-      return requireModule(path.resolve(fullpath));
-    }
-
     if (err.code !== 'ERR_REQUIRE_ESM') {
       throw err;
     }
 
-    return import(fullpath).then(result => (result.default || {}));
+    return import(fullpath).then(result => result.default || {});
```

Comment:
can you revert this change? I know it's from the automatic lint fixes but the extra parentheses help provide better visibility

Reply:
Done